src/options.ts

```diff
@@ -6,6 +6,7 @@ export interface IVPCOptions {
     vpcId: string;
     securityGroupIds: string[];
     subnetIds: string[];
+    elbSubnetIds: string[];
```

Comment:
Sorry for taking to long. About to merge, could you fix this, please? I believe this needs to be optional. `elbSubnetIds?: string[];`

Reply:
thanks. fixed it